src/modulemgr/modulemgr.c

```diff
@@ -309,27 +309,21 @@ static s32 exe_thread(SceSize args, void *argp)
 }
 
 // 0x0000501C
-s32 ModuleMgrRebootPhase(SceSize argc, void *argp)
+s32 ModuleMgrRebootPhase()

```

Comment:
Similar to `module_reboot_before`, for kernel module at least it takes 4 arguments (it looks like an equivalent reboot_phase entry point does ot exist for user modules): https://github.com/uofw/uofw/blob/276cb776451721f77775d71bee9d60bf17b07ba3/src/modulemgr/rebootModule.c#L149

Now, I originally gave it the same signature as `module_reboot_before` (the kernel variant of it): 
```c++
typedef s32 (*SceKernelRebootKernelThreadEntry)(s32 arg1, u32 arg2, s32 arg3, s32 arg4);
```

This, however, seems incorrect based on calls made in the `Loadexec` module. Here's a call to `sceKernelRebootBeforeForKernel()`:
https://github.com/uofw/uofw/blob/276cb776451721f77775d71bee9d60bf17b07ba3/src/loadexec/loadexec.c#L1530

And here's a call to `sceKernelRebootPhaseForKernel()`:
https://github.com/uofw/uofw/blob/276cb776451721f77775d71bee9d60bf17b07ba3/src/loadexec/loadexec.c#L1493

So the actual function signatures to use here should actually be like this:
```c++
typedef s32 (*SceKernelRebootBeforeForKernel)(void *arg1, s32 arg2, s32 arg3, s32 arg4); // the current "SceKernelRebootKernelThreadEntry"
```

and
```c++
typedef s32 (*SceKernelRebootPhaseForKernel)(s32 arg1, void *arg2, s32 arg3, s32 arg4);
```

Those definitions would have to be updated at least here:
https://github.com/uofw/uofw/blob/276cb776451721f77775d71bee9d60bf17b07ba3/include/loadcore.h#L749
and here:
https://github.com/uofw/uofw/blob/276cb776451721f77775d71bee9d60bf17b07ba3/include/loadcore.h#L754

Reply:
Addressed in 13f82c3531df835cc9774e8e97f0e715bb3b6a53